pmgr/CfgModel.py

```diff
@@ -232,8 +228,15 @@ def db2index(self, idx, f):
         except:
             return None
 
-    def getCfg(self, idx, loop=[]):
-        if idx == None or idx in loop:
+    #
+    # getCfg(idx) - Get the configuration dictionary for this index.  Negative
```

Comment:
Just a general thing, you might consider moving all these pre-`def` comments to docstrings inside of their related functions. Could be used to generate API docs in the future.

Reply:
I'll do that now.  Not sure what do do with the comments describing the internal variables... I'll probably just leave them as comments in the code, unless I hear suggestions otherwise.

Reply:
Code and refactor suggestions are always welcome. ;-)